applovin_max/android/src/main/java/com/applovin/applovin_max/AppLovinMAX.java

```diff
@@ -199,8 +210,34 @@ private void initialize(final String pluginVersion, final String sdkKey, final R
             }
         }
 
+        AppLovinSdkSettings settings = new AppLovinSdkSettings( applicationContext );
```

Comment:
Same as iOS, let's update the other settings before we pass the settings object to the SDK.